README.md

```diff
@@ -110,9 +103,13 @@ After successfully bootstrapping your cluster, you are ready to start the packag
 glasskube serve
 ```
 
-This command will open [`http://localhost:8580`](http://localhost:8580) in your default browser. 
+This command will open [`http://localhost:8580`](http://localhost:8580) in your default browser.
 Congratulations, you can now explore and install all our available packages! 🎉
 
+## 🎬 How to install you first package
+
+![cast](https://github.com/glasskube/glasskube/assets/16959694/f8b936ca-7b58-4e2b-8845-17da089f2384)
```

Comment:
This is starting to feel out of place, in addition to being a seriously outdated demo. IMO we should remove it and make coming up with an alternative a priority.

Reply:
@jakepage91 is already working on the alternative - as soon as it is ready I will exchange it in a follow up pr.